content/04.network_applications.md

```diff
@@ -31,11 +31,34 @@
 
 1. Mention predicting genes interacting genes
 
-#### Drug Interactions
-
-1. Talk about drug side effects
-2. Drug repurposing
-3. Drug-Disease Interations
+#### Pharmaceutical Applications
+
+Pharmaceutical applications consist of using knowledge graphs to discover new information on drugs.
+Tasks in this context involve prediction drugs interacting with other drugs [@doi:10.1016/j.websem.2017.06.002], identifying molecular targets a drug might interact with [@doi:10.1155/2015/275045] and identifying new disease treatments for previously established drugs [@doi:10.7554/eLife.26726.001].
+We discuss pharmaceutical applications that project knowledge graph onto a low dimensional space.
+
+Similar to multi-omic applications knowledge graphs have been used in recommendation systems to infer novel information between drugs and diseases.
+Dai et al. [@doi:10.1155/2015/275045] used collaborative filtering to infer drug-disease associations.
+The authors constructed a drug-disease network by integrating two bipartite networks: a drug-gene interaction network and a disease-gene interaction network.
+Ultimately most models utilize the popular word2vec variants to embed knolwedge graphs into a low dimensional space; however, translational models have been under utilized in these approaches which means people should work on incorporating more.
+They integrated both networks under the assumption that a drug is associated with a disease if they interact with the same gene of interest. 
```

Comment:
```suggestion
They integrated both networks under the assumption that a drugs associated with a disease interact with the same gene of interest. 
```
I didn't understand the meaning of this sentence so I've revised it to something I do understand. You should check this to make sure it's what you intend though, since I'm not sure it's correct :)